### DISCUSSION TITLE
Question wrt ChaosStrikesBack&EnglishUtilityDisk_v1.21.lha

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1550] [Q&A]
File in question: http://ftp2.grandis.nu/turran/FTP/Retroplay%20WHDLoad%20Packs/Commodore_Amiga_-_WHDLoad_-_Games/C/ChaosStrikesBack&EnglishUtilityDisk_v1.21.lha

After direct-loading this title, the whdload splashscreen discloses...

![ex](https://imgur.com/FYR85yS.png)

...but I can't get the custom options to work.

The question is, what's wrong here?....

1. The XML snippet for this title, contains no custom options.... //might be fodder for @HoraceAndTheSpider 
2. The whdload.lha file is being incorrectly described.... //mistakes happen, or, it expects the below to work
3. The GUI -> WHDLoad --> Custom: stringbox isn't working... //me suspecting this situation is the reason for this stringbox

Any idea what's the correct answer here?

TIA

**Comment by midwan** [comment]
Not sure, this looks like a bit of a special case.
@HoraceAndTheSpider what do you think?
